### PR TITLE
FE App changes

### PR DESCRIPTION
**Pull request by arshDevslane**

### CODE REVIEW

File: src/services/api-services/BaseApiService.ts [hunk 1]
```diff
@@ -89,12 +90,21 @@ export class BaseApiService {
       };
     }
   ) {
+    const payload = {
+      id,
+      ver: '1.0',
+      ts: '2024-09-20T17:43:49+05:30',
```
Comment (HarishGangula) [inline src/services/api-services/BaseApiService.ts:96]: Don't hard code this value

File: src/services/api-services/BaseApiService.ts [hunk 2]
```diff
@@ -89,12 +90,21 @@ export class BaseApiService {
...
+      params: {
+        msgid: 'd297e367-7b72-45c1-883c-18005a7ea888',
```
Comment (HarishGangula) [inline src/services/api-services/BaseApiService.ts:98]: This should dynamic uuid

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: src/services/api-services/BaseApiService.ts (hunk 1 above)
Comment (arshDevslane) [reply src/services/api-services/BaseApiService.ts:96]: Yes for the moment i hardcoded it, it's been fixed now 👍

File: src/services/api-services/BaseApiService.ts (hunk 2 above)
Comment (arshDevslane) [reply src/services/api-services/BaseApiService.ts:98]: this too has been fixed

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**